### ISSUE TITLE
Globalization Extension is biased against English characters

### ISSUE THREAD

**Issue by mhmd-azeez** [xoofx/markdig#619]
Due to this optimization, the extension ignores English characters entirely. So for this text:
```
aaa سڵاو
```
expected: LTR
actual: RTL

And this case:
```
سڵاو aaa
```
expected: RTL
actual: RTL

(This case above is correct, I just wrote it to make sure we don't mess it up when tying to fix the first one).

The fix is just to remove [this check](https://github.com/xoofx/markdig/blob/cb26f30f7bff7dd91bc69902a80cf97f0d445b69/src/Markdig/Extensions/Globalization/GlobalizationExtension.cs#L96).

**Comment by MihaZupan** [comment]
You're right, nice catch. Looks like I broke this in c964659085ec53ce27e7044e105303e0fba8ee36.

**Comment by mhmd-azeez** [comment]
@MihaZupan no worries, it happens. I am surprised none of the tests broke though
